R/orcutt-tidiers.R

```diff
@@ -28,12 +28,22 @@
 #' @export
 #' @family orcutt tidiers
 #' @seealso [orcutt::cochrane.orcutt()]
-tidy.orcutt <- function(x, ...) {
-  warning("deal with tidy.orcutt conf.int nonsense")
-  dots <- enquos(...)
-  dots$conf.int <- FALSE
+tidy.orcutt <- function(x, exponentiate = FALSE, ...) {
+  s <- summary(x)
+  ret <- tidy.summary.orcutt(s)
+  
+  process_orcutt(ret, x, exponentiate = exponentiate)
+}
 
-  rlang::exec(tidy.lm, x, !!!dots)
+#' @rdname orcutt
+#' @export
+tidy.summary.orcutt <- function(x, ...) {
```

Comment:
As a general rule I prefer to have a single `tidy()` method that works on the original object, without an extra `tidy()` method for the summary. Tidying a summary feels like double-tidying to me, and I don't really know how to think about it.